lecture_notes/lesson_3.md

```diff
@@ -73,6 +77,19 @@ if 2 < x < 4:
     print("x is between 2 and 4")    
 ```
 
+## Terminating a program
+* There are cases where we need to terminate program earlier because there is no need to process anything further.
+* To exit a program we can use the function `exit()` or `quit()`, with the parameter that determines the exit code (we do not cover exit codes in this course).
+* Note: It's recommended to use `sys.exit()` in programs. The official Python [documentation](https://docs.python.org/3/library/sys.html#sys.exit) recommends `exit()` and `quit()`only for interactive interpreter shell, but for the sake of fast learning we'll use these functions also in programs.
```

Comment:
Tady bych dal jenom zmínku, že to doporučení je a link. 
'interactive interpreter shell' je podle mě moc abstraktní na to, aby si pod tím něco představili a uvědomili si smysl

Reply:
Můžu to odmazat. Btw v 1. lekci jim vysvětlujeme rozdíl mezi interaktivním x neinteraktivním režimem.